index.html

```diff
@@ -6,17 +6,60 @@
       name="viewport"
       content="width=device-width, initial-scale=1, shrink-to-fit=no"
     />
-    <title>My Blog</title>
     <link
       href="//fonts.googleapis.com/css?family=Roboto:400,500,300"
       rel="stylesheet"
       type="text/css"
     />
     <link rel="stylesheet" href="css/style.css" />
     <link rel="shortcut icon" type="image/x-icon" href="favicon.ico" />
+    <title>Mathsli</title>
   </head>
   <body>
-    <!-- Add your HTML markup here -->
-    <!-- Remember: Use semantic HTML tags like <header>, <main>, <nav>, <footer>, <section> etc -->
+    <header>
+     <h1>Mathsli</h1> 
+     <h3>Do you love Maths? </h3>
+     <h5>Or Maybe...?</h5>
+     <p>Here is the right place for you to be anyway.</p>
+     <nav>
+      <ul class="list">
+          <li class="items"><a class="links" href="#"> About Mathli</a></li>
+          <li class="items"><a class="links" href="#"> Testimonials</a></li>
+          <li class="items"><a class="links" href="#"> Contact</a></li>
+          <li class="items"><a class="links" href="#"> Login</a></li>  
+      </ul>     
+  </nav>
+    </header>
+    <main>
+      <section>
+      <article class="article1">
+        <h1>About Mathsli</h1>
```

Comment:
There should only be one `h1` on each page. Use it to wrap the title that best describes what the page is about